tests/meta/upload/test_upload_api.py

```diff
@@ -3,12 +3,17 @@
 import datetime as dt
 
 from cg.meta.upload.upload_api import UploadAPI
-from cg.models.cg_config import CGConfig
 from cg.store.models import Analysis, Case
 from tests.cli.workflow.conftest import tb_api
+from tests.cli.workflow.mip.conftest import (
+    mip_case_id,
+    mip_case_ids,
+    mip_dna_context,
+    mip_rna_context,
+)
 
 
-def test_mip_dna_update_uploaded_at(mip_dna_context: CGConfig, mip_dna_case: Case):
+def test_mip_dna_update_uploaded_at(mip_dna_context: mip_dna_context, mip_dna_case: Case):
```

Comment:
The previous type hint was correct
```suggestion
def test_mip_dna_update_uploaded_at(mip_dna_context: CGConfig, mip_dna_case: Case):
```